File: src/main/resources/static/js/services/associadoService.js
angular.module("FuneSysApp").factory("AssociadoService", function($http, config){
	var _getAssociados = function(){
		return $http.get(config.baseUrl + "/associado/findall");
	};

	var _saveAssociado = function(associado){
		return $http.post(config.baseUrl + "/associado/save", associado);
	};

	var _deleteAssociado = function(associado){
		return $http.post(config.baseUrl + "/associado/delete", associado);
	};

	return{
		getAssociados: _getAssociados,
		saveAssociado: _saveAssociado,
		deleteAssociado: _deleteAssociado
	};
});

File: src/main/java/br/edu/estacio/pos/service/AssociadoService.java
package br.edu.estacio.pos.service;

import java.util.List;

import org.assertj.core.util.Lists;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import br.edu.estacio.pos.model.Associado;
import br.edu.estacio.pos.repository.AssociadoRepository;

@Service
public class AssociadoService implements IAssociadoService {

	@Autowired
	private AssociadoRepository repository;
	
	@Override
	public Associado save(Associado associado) {
		return repository.save(associado);
	}

	@Override
	public Associado findOne(Integer id) {
		return repository.findOne(id);
	}

	@Override
	public List<Associado> findAll() {
		return Lists.newArrayList(repository.findAll());
	}

	@Override
	public void delete(Associado associado) {
		repository.delete(associado.getId());
	}

}
